src/add-text-track-data.js

```diff
@@ -99,11 +99,9 @@ const addTextTrackData = function(sourceHandler, captionArray, metadataArray) {
       for (let i = 0; i <= sourceHandler.metadataTrack_.cues.length - 1; i++) {
```

Comment:
This is better written as `for (let i = 0; i < sourceHandler.metadataTrack_.cues.length; i++) {`. Most developers are used to this and will automatically read it as "for each element in the array".